docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.row.scss

```diff
@@ -76,6 +76,10 @@
   }
 }
 
+.counted-items-1.field-hs-row-components {
```

Comment:
I have one question about this scss style -- why do you add the classes on the same line, rather than adding an &.field-hs-row-components nested below? I was just curious and I want us to sync up on a standard style strategy

Reply:
I didn't nest it since it's such a simple rule and it didn't feel necessary to break up the styles. If there were more styles or more things nested under the `.counted-items-1` class then I would have nested like you mentioned. Both ways of writing the styles are correct and give the same results, it's just a matter or preference.